metadata.json

```diff
@@ -12,21 +12,16 @@
       "operatingsystem": "RedHat",
       "operatingsystemrelease": [
         "6",
-        "7"
-      ]
-    },
-    {
-      "operatingsystem": "CentOS",
```

Comment:
We still support CentOS. I don't see a reason to drop this.

Reply:
Whoops, that wasn't intentional.  Should CentOS 8 be listed in the metadata?  I know it's not out yet but adding it to the OS list now would avoid having to update metadata in the future.

Reply:
Also, Fedora 28 is now EOL.  Should that be removed?

Reply:
Yes, I think we should list CentOS 8. If there's an issue, it'd be a bug we need to fix. We can also reasonably assume that when it works on RHEL 8, it'll work on CentOS 8.

I'd leave F28 in for now because it can be seen as a breaking change and I've done enough of those for now ;) There's no additional code needed so there's little benefit.

Reply:
Okay, I've updated this branch to include CentOS 8 along with RHEL 8 and Fedora 30.